app/uk/gov/hmrc/cgtpropertydisposals/controllers/SubscriptionController.scala

```diff
@@ -52,7 +56,12 @@ class SubscriptionController @Inject()(service: SubscriptionService, cc: Control
                                       RequestValidationError(s"Could not parse JSON body as subscription request: $e")
                                   )
                               )
-        subscriptionResponse <- service.subscribe(subscriptionDetails).leftMap[SubscriptionError](BackendError(_))
+        subscriptionResponse <- subscriptionService
+                                 .subscribe(subscriptionDetails)
+                                 .leftMap[SubscriptionError](BackendError(_))
+        _ <- taxEnrolmentService
```

Comment:
if the subscription fails we still want to see the subscription confirmation page. What will happen at the moment is that an error page will be shown since errors are mapped to a `BackendError` in the `Left`.

Reply:
refactored to return 200 OK even if enrolment call fails